File: GaoDeDemo/src/main/java/com/tianshaokai/gaodedemo/MainActivity.java
package com.tianshaokai.gaodedemo;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.amap.api.maps.MapView;

public class MainActivity extends AppCompatActivity {

    MapView map;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        map = findViewById(R.id.map);

        //在activity执行onCreate时执行mMapView.onCreate(savedInstanceState)，创建地图
        map.onCreate(savedInstanceState);
    }
}

File: uiapp/src/main/java/com/tianshaokai/studyuiapp/MainActivity.kt
package com.tianshaokai.studyuiapp

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        text.setOnClickListener(View.OnClickListener {
            DbActivity.lanchMode(this)
        })
    }
}
